.github/workflows/release-porting-guide.yml

```
@@ -0,0 +1,61 @@
+---
+name: Ansible porting guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: Release version. For example, 11.1.0
+        required: true
+      ansible-major-version:
+        description: Major version. For example, 11
+        required: true
+      ansible-build-data-branch:
+        description: Release Branch name from the Ansible Build data PR
```

I think this should give an example of a ref to an ephemeral PR merge commit
```suggestion
        description: >-
          Release Branch name from the Ansible Build data PR.
          (e.g. `refs/pull/1/merge`)
```
because who knows if the PR branch is always created from upstream and not some fork, and also, we probably want things "as merged".